### ISSUE TITLE
hcdev web should be able to watch for changes

### ISSUE THREAD

**Issue by thedavidmeister** [holochain/holochain-proto#648]
currently it seems i need to down and up the web server to see changes to my zomes during dev

**Comment by Connoropolous** [comment]
yeah, it's a pain... I documented a workaround you can use, while we wait for it to find its way into the main code (mac only, although finding a linux alternative should be really simple)

https://developer.holochain.org/webserver_auto_restart

**Comment by thedavidmeister** [comment]
yeah i eventually found this, cheers!

**Comment by thedavidmeister** [comment]
the main pain for plugging it into existing setups that watch seems to be the manual kill/restart process but a `-w` flag or something might be nice too

**Comment by JhonatanHern** [comment]
I made a little script to solve that issue:

[holo-server.js](https://github.com/JhonatanHern/holo-casting/blob/master/backend/holo-server.js)

It works really fine. In order to run it you have to run:

```
npm i chokidar
```
and then you run the program like this:
```
node holo-server.js
```
all inside the main folder of the holochain project.
